tasking/pulp/tasking/services/scheduler.py

```diff
@@ -9,15 +10,20 @@
 from django.utils import timezone
 from django.db import IntegrityError
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", 'pulp.app.settings')
+
+# This import is here so that Celery will find our application instance. It's important that other
+# pulp and django code not get used until after the celery app is instantiated and does its "fixup"
+# of django.
```

Comment:
That makes sense! I've changed the order to the usual convention and everything still works.